mgmtworker/mgmtworker/monitoring.py

```diff
@@ -1,77 +1,117 @@
 import json
 
-from os import sep
+from os import rename, sep
 from os.path import join
 
+from jinja2 import Template
+from tempfile import NamedTemporaryFile
+
 from cloudify.utils import setup_logger
 
 logger = setup_logger('cloudify.monitoring')
 
 PROMETHEUS_CONFIG_DIR = join(sep, 'etc', 'prometheus', )
 PROMETHEUS_TARGETS_DIR = join(PROMETHEUS_CONFIG_DIR, 'targets')
+PROMETHEUS_ALERTS_DIR = join(PROMETHEUS_CONFIG_DIR, 'alerts')
 PROMETHEUS_TARGETS_TEMPLATE = '- targets: {target_addresses}\n'\
                               '  labels: {target_labels}'
+PROMETHEUS_MISSING_ALERT = Template(
+    """groups:
+  - name: {{ name }}
+    rules:
+
```

Comment:
Why the blank line?

Reply:
Note that this gives output like:
`groups:
  - name: test
    rules:


      - alert: test_missing
        expr: absent(test_healthy{host="host1"})
        for: 1m
        labels:
          severity: critical
        annotations:
          summary: "Test is missing on node host1"

      - alert: test_missing
        expr: absent(test_healthy{host="host2"})
        for: 1m
        labels:
          severity: critical
        annotations:
          summary: "Test is missing on node host2"

`

Reply:
Note that this gives output like:

> groups:
>  - name: test
>    rules:
>
>
>     - alert: test_missing
>        expr: absent(test_healthy{host="host1"})
>        for: 1m
>        labels:
>          severity: critical
>        annotations:
>          summary: "Test is missing on node host1"
>
      - alert: test_missing
        expr: absent(test_healthy{host="host2"})
        for: 1m
        labels:
          severity: critical
        annotations:
          summary: "Test is missing on node host2"